### PR TITLE
CORTX-34395: Update docker-compose installation and nfs mount role

### PR DESCRIPTION
**Pull request by gauravchaudhari02**
# Problem Statement
- docker-compose and nfs mount were not added as a part of prerequisites installation playbooks.

# Design
-  Added tasks for docker-compose installation and setup required nfs mount

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [x] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [ ] Jenkins pipelines used for testing

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] Jira ID/ COMMUNITY is prefixed in PR title
      Prefix Syntax - 
        `CORTX-<5 digit JIRA ID>: <PR Title>` - for Seagate Employees
        `COMMUNITY: <PR Title>`  - for open source contributors
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [x] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by pankajgborole — DISMISSED** [review]
Approved

**Review by shailesh-vaidya — APPROVED** [review]

**Review by pankajgborole — DISMISSED** [review]
Approved

File: solutions/vmware/migration/ansible/vm-metadata-operations/roles/setup_prerequisites/tasks/install_docker.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    url: https://download.docker.com/linux/centos/docker-ce.repo
+    dest: /etc/yum.repos.d/docer-ce.repo
+
+- name: Install Docker and Docker Compose
+  yum:
+    name: 
+      - "docker-ce-{{ docker_version }}"
+      - "docker-ce-cli-{{ docker_version }}"
+      - "containerd.io"
+      - "docker-compose-plugin"
```
Comment (shailesh-vaidya) [inline solutions/vmware/migration/ansible/vm-metadata-operations/roles/setup_prerequisites/tasks/install_docker.yml:18]: @gauravchaudhari02 - `docker-compose-plugin` package does not install `docker-compose` command which we hase used in scripts. Refer - https://github.com/Seagate/cortx-re/blob/main/docker/cortx-deploy/build.sh#L126 

```bash
[root@ssc-cicd-4094 ~]# yum list docker-compose-plugin
Updating Subscription Management repositories.
EPEL 8 - Everything                                                                                                                                       45 kB/s | 2.3 kB     00:00
Rocky Linux 8 - AppStream                                                                                                                                 53 kB/s | 2.6 kB     00:00
Docker-CentOS-8                                                                                                                                           38 kB/s | 2.0 kB     00:00
Rocky Linux 8 - PowerTools                                                                                                                                50 kB/s | 2.6 kB     00:00
puppet6-el8                                                                                                                                               34 kB/s | 1.7 kB     00:00
Rocky Linux 8 - Extras                                                                                                                                    41 kB/s | 2.0 kB     00:00
Rocky Linux 8 - BaseOS                                                                                                                                    44 kB/s | 2.3 kB     00:00
Rocky Linux 8 - ResilientStorage                                                                                                                          46 kB/s | 2.3 kB     00:00
Kubernetes-EL7                                                                                                                                            43 kB/s | 2.0 kB     00:00
Installed Packages
docker-compose-plugin.x86_64                                                                2.14.1-3.el8                                                                @docker-ce-stable
[root@ssc-cicd-4094 ~]# docker-compose
-bash: docker-compose: command not found
[root@ssc-cicd-4094 ~]#
```

We either need to update scripts or use https://docs.docker.com/compose/install/linux/#install-the-plugin-manually

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

**Review by shailesh-vaidya — APPROVED** [review]

File: solutions/vmware/migration/ansible/vm-metadata-operations/roles/setup_prerequisites/tasks/install_docker.yml (hunk 1 above)
Comment (gauravchaudhari02) [reply solutions/vmware/migration/ansible/vm-metadata-operations/roles/setup_prerequisites/tasks/install_docker.yml]: Added suggested way to install docker-compose

**Review by pankajgborole — APPROVED** [review]
Approved

**Pull request merged**